.github/workflows/smart-integ-inferno-test-periodically.yaml

```diff
@@ -3,6 +3,10 @@ on:
   schedule:
     - cron: '0 13 1,15 * *'
 
+env:
```

Comment:
you may need to pull changes to revert these: we pivoted away from using AWS_ACCESS_KEY_ID in the secrets and instead use the github action for sts short-term credentials.